src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -7,226 +7,217 @@
 using System.Linq;
 using Microsoft.CodeAnalysis.AddImports;
 using Microsoft.CodeAnalysis.CodeStyle;
-
-#if CODE_STYLE
-using Microsoft.CodeAnalysis.Internal.Options;
-#else
 using Microsoft.CodeAnalysis.Options;
-#endif
 
-#if CODE_STYLE
-namespace Microsoft.CodeAnalysis.CSharp.Internal.CodeStyle
-#else
 namespace Microsoft.CodeAnalysis.CSharp.CodeStyle
-#endif
 {
     internal static partial class CSharpCodeStyleOptions
```

Comment:
`CSharpCodeStyleOptions` was already an internal type, which we had moved into shared layer, so just needed changes to moved to new options types.